src/main/resources/com/mathworks/ci/BuildArtifactAction/index.jelly

```diff
@@ -23,20 +22,20 @@
                             (${h.getDiffString(it.failCount-prev.failCount)})
                         </j:if>
                         <j:if test="${it.skipCount > 0}">
-                            , Skipped ${(it.skipCount)}
+                            , ${(it.skipCount)} Skipped
```

Comment:
Is the leading comma intended?

Reply:
It acts as a separator between Total tasks count and skipped tasks count. Although if you check the [above image](https://github.com/mathworks/jenkins-matlab-plugin/pull/356#issuecomment-2316828981) there's an extra space before the comma, do you know how we can fix that?